x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/policy_template_input_selector.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import {
+  EuiFlexGroup,
+  EuiToolTip,
+  EuiFlexItem,
+  EuiIcon,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { css } from '@emotion/react';
+import type { PostureInput, PosturePolicyTemplate } from '../../../common/constants';
+import { getPolicyTemplateInputOptions, type NewPackagePolicyPostureInput } from './utils';
+import { InlineRadioGroup } from './inline_radio_group';
+
+type Props = { input: NewPackagePolicyPostureInput } & (
+  | { disabled: true }
+  | {
+      disabled?: false;
+      updatePolicyInput(id: string): void;
+    }
+);
+
+const noop = () => {};
```

Comment:
this can be removed, but we still need to provide `onChange` for types. 

in anyway, not calling a disabled option is already handled by `eui` as we're technically using the original `EuiRadioGroup` component

Reply:
removed